uberfire-workbench/uberfire-workbench-client/src/main/java/org/uberfire/client/workbench/PanelManager.java

```diff
@@ -217,4 +217,9 @@ void setRoot(PerspectiveActivity activity,
      * @param part the part that has been minimized
      */
     void onPartMinimized(PartDefinition part);
+
+    /**
+     * @return the focused {@link PanelDefinition}
```

Comment:
`PartDefinition` instead of `PanelDefinition`?

Reply:
Good catch!

Reply:
Fixed! Thank you, @jomarko :-)